www/src/scss/_overlay.scss

```diff
@@ -61,6 +61,30 @@
     width: auto;
   }
 }
+.url-container {
+  display: inline-block;
+  margin-top: 1vh;
+  padding: 8px;
```

Comment:
this element should have either padding or margin (preferably margin). Let stick with one, use `margin: 9px` maybe?